deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1930,14 +1934,16 @@ extension ChatViewController {
                     UIAction.menuAction(localizationKey: "forward", image: image, indexPath: indexPath, action: forward)
                 )
 
-                if dcChat.isSelfTalk || message.savedMessageId != 0 {
-                    children.append(
-                        UIAction.menuAction(localizationKey: "unsave", systemImageName: "star.slash", indexPath: indexPath, action: toggleSave)
-                    )
-                } else {
-                    children.append(
-                        UIAction.menuAction(localizationKey: "save_desktop", systemImageName: "star", indexPath: indexPath, action: toggleSave)
-                    )
+                if !message.isInfo { // info-messages out of context results in confusion, see https://github.com/deltachat/deltachat-ios/issues/2567
```

Comment:
Does this mean that you can save marker messages like video invitations? I guess, this is intended?

Reply:
did not thought about that. but better use the same isMarkerOrInfo() condition as above for now. 

might otherwise be more confusion than helpful -  they all look like info-messages, so it seems fine if they work similiar. also, these messages maybe need additional cases for displaying the "star" etc. on other platforms, so, for now, just don't allow to save them

Reply:
i updated the condition